### PR TITLE
Certificate management

### PR DESCRIPTION
**Pull request by cadecairos**
@alicoding @gideonthomas R?

### CODE REVIEW

**Comment by gideonthomas** [comment]
Is there a way to install the cert on AWS through the interface? If so, can you document that here (some of us prefer to use the interface so it would be nice to know how to do it through there as well)

File: docs/aws/ssl-certificate-management.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+17. Agree to the TOS presented at the bottom of the form. Or don't. depends how badly you need that cert!
+18. Submit the Certificate Request.
+19. Chris, and possibly you (TBD), will receive an email asking to confirm the request. It should contain a link to the request on the Digicert website, where you should select the approval button.
+20. The signed certificate for the domain will be emailed to you! extract the contents of the attachment to the same directory we created the key and CSR in.
+
+### Install the certificate
+
+#### On AWS
+
+1. Ensure you've configured the [AWS cli](http://docs.aws.amazon.com/cli/latest/userguide/installing.html)for your machine
```
Comment (alicoding) [inline docs/aws/ssl-certificate-management.md]: space before `for`

**Comment by cadecairos** [comment]
> Is there a way to install the cert on AWS through the interface? If so, can you document that here (some of us prefer to use the interface so it would be nice to know how to do it through there as well)

@gideonthomas there's no interface to do that.

**Comment by gideonthomas** [comment]
r+

**Pull request merged**